test_regress/driver.pl

```diff
@@ -133,6 +134,11 @@ BEGIN
 $opt_jobs = calc_jobs() if defined $opt_jobs && $opt_jobs == 0;
 $Fork->max_proc($opt_jobs);
 
+my $use_debugger = $opt_gdb || $opt_gdbbt || $opt_gdbsim || $opt_rr || $opt_rrsim;
```

Comment:
```suggestion
my $interactive_debugger = $opt_gdb || $opt_gdbsim || $opt_rr || $opt_rrsim;
```

gdbbt is not interactive.